### PR TITLE
tests: add details to regression tests

### PR DESCRIPTION
**Pull request by zyga**

### CODE REVIEW

File: tests/regression/lp-1852361/task.yaml [hunk 1]
```diff
@@ -1,5 +1,22 @@
 summary: regression test for https://bugs.launchpad.net/snapd/+bug/1852361
 
+details: |
+    Snapd attempts to update preserved mount namespaces when the per-snap mount
```
Comment (sergiocazzolato) [inline tests/regression/lp-1852361/task.yaml:4]: This it the same description than tests/regression/lp-1844496/task.yaml

Comment (sergiocazzolato) [reply tests/regression/lp-1852361/task.yaml:4]: is this one cased by the same issue?

**Review by sergiocazzolato — APPROVED** [review]
Awesome, thanks

File: tests/regression/lp-1852361/task.yaml (hunk 1 above)
Comment (zyga) [reply tests/regression/lp-1852361/task.yaml:4]: Yes, they are all cross-linked in the last paragraph.

**Review by bboozzoo — APPROVED** [review]
LGTM

**Pull request merged**